package.json

```diff
@@ -2,9 +2,10 @@
   "name": "electron-pdf",
   "version": "0.12.0",
   "description": "A command line tool to generate PDF from URL, HTML or Markdown files",
-  "main": "index.js",
+  "main": "lib/index.js",
   "scripts": {
-    "test": "standard"
+    "lint": "standard",
+    "test": "standard && node_modules/.bin/electron node_modules/.bin/jasmine"
```

Comment:
I've made some changes such that the index.js is expected to be run inside of a electron process, so therefore the tests have to be run with electron.
I had `standard && electron jasmine` but that did not work.  `standard && jasmine node_modules/.bin/jasmine` worked for me locally though.  Neither work with Travis :(